anitya/templates/project.html

```diff
@@ -191,6 +191,10 @@ <h3 class="panel-title">Mappings</h3>
                       <a href="https://src.fedoraproject.org/rpms/{{ package.package_name }}">
                         {{ package.package_name }}
                       </a>
+                    {% elseif package.distro_name == 'PLD-Linux' %}
```

Comment:
I'm not sure what template engine is used here, so this may not work.

Reply:
We are using jinja2 as template engine. I will try this on my local instance of Anitya to make sure, that it's working.

Reply:
Note to myself, using jinja2:
- https://github.com/fedora-infra/anitya/blob/215adea038141bb59a4dd825e336b09d2c96f26e/requirements.txt#L8

Reply:
@Zlopez no need to test locally, Travis CI caught the invalid syntax fine:
- https://travis-ci.org/github/fedora-infra/anitya/jobs/763250533#L1331

Reply:
I still want to see that the link is really working.